org.eclipse.winery.workflowmodeler/src/app/services/winery.service.ts

```diff
@@ -0,0 +1,175 @@
+        });
+    }
+
+    public loadTopologyProperties(nodeTemplate: NodeTemplate) {
+        const url = 'nodetypes/' + this.encode(nodeTemplate.namespace)
+            + '/' + this.encode(nodeTemplate.id) + '/propertiesdefinition/winery/list/';
+
+        this.request(url, properties => {
+            properties.forEach(property => nodeTemplate.properties.push(property.key));
+        });
+    }
+
+    public loadNodeTemplateInterfaces(namespace: string, nodeType: string) {
+        const url = 'nodetypes/' + this.encode(namespace)
+            + '/' + this.encode(nodeType) + '/interfaces/';
+        const wineryService = this;
+        this.request(url, interfaces => {
+            wineryService.broadcastService.broadcast(wineryService.broadcastService.nodeInterfaces, interfaces);
+        });
+    }
+
+    public loadNodeTemplateOperations(namespace: string, nodeType: string, interfaceName: string) {
+        const wineryService = this;
+        const url = 'nodetypes/' + this.encode(namespace)
+            + '/' + this.encode(nodeType) + '/interfaces/' + this.encode(interfaceName) + '/operations/';
+
+        this.request(url, operations => {
+            const res = [];
+            operations.forEach(operation => res.push(new Operation(operation)));
+            wineryService.broadcastService.broadcast(wineryService.broadcastService.nodeOperations, res);
+        });
+    }
+
+    public loadNodeTemplateOperationParameter(namespace: string, nodeType: string, interfaceName: string, operation: string) {
+        const wineryService = this;
+        const params = {
+            input: [],
+            output: [],
+        };
+
+        // inputparameters
+        const relativePath = 'nodetypes/' + this.encode(namespace) + '/' + this.encode(nodeType)
+            + '/interfaces/' + this.encode(interfaceName) + '/operations/' + this.encode(operation) + '/';
+        this.request(relativePath + 'inputparameters', response => {
+            params.input = response;
+            wineryService.broadcastService.broadcast(wineryService.broadcastService.nodeParameters, params);
+        });
+
+        // outputparameters
+        this.request(relativePath + 'outputparameters', response => {
+            params.output = response;
+            wineryService.broadcastService.broadcast(wineryService.broadcastService.nodeParameters, params);
+        });
+    }
+
+    public save(data: string) {
+        const url = 'servicetemplates/' + this.encode(this.namespace)
+            + '/' + this.encode(this.serviceTemplateId) + '/plans/' + this.encode(this.plan) + '/file';
+
+        const requestData = '-----------------------------7da24f2e50046\r\n'
+            + 'Content-Disposition: form-data; name=\"file\"; filename=\"file.json\"\r\n'
+            + 'Content-type: plain/text\r\n\r\n'
+            + data + '\r\n-----------------------------7da24f2e50046--\r\n';
+
+        $.ajax({

```

Comment:
Yep, I'm trying to replace jquery.ajax to angular.http. And the other places will be deleted too.